src/wasm/literal.cpp

```diff
@@ -147,7 +156,6 @@ Literal::Literal(const Literal& other) : type(other.type) {
         case HeapType::cont:
         case HeapType::struct_:
         case HeapType::array:
-        case HeapType::exn:
           WASM_UNREACHABLE("invalid type");
         case HeapType::string:
           WASM_UNREACHABLE("TODO: string literals");
```

Comment:
About 15 lines below here, there is the destructor `~Literal`. It needs code to destroy `exnData`, parallel to `gcData`.

Reply:
Oops, thanks. I've lived to long among languages with a GC; I lost some reflexes. :laughing: 

Fixed.